src/dynamic_loading_c_impl.rs

```diff
@@ -159,6 +163,19 @@ impl<T> CKBDLContext<T> {
     }
 
     pub fn load<'a>(&'a mut self, dep_cell_data_hash: &[u8]) -> Result<Library, Error> {
```

Comment:
I think we should mark this function as deprecated, then add a new one maybe `load_by_data`, that let users choose among `Data`, `Data1`, etc. It is the same behavior now, does not mean we should take it for granted. Also, we might want to fix the internal logic, so errors will be thrown if current VM is running in `data1` mode, and the contract tries to load a contract via `data`, or vice versa.